packages/deparser/src/deparser.js

```diff
@@ -283,12 +302,12 @@ export default class Deparser {
       return null;
     }
 
-    if (_.isNumber(item)) {
+    if (typeof item === 'number' || item instanceof Number) {
       return item;
     }
 
-    const type = keys(item)[0];
-    const node = _.values(item)[0];
+    const type = Object.keys(item)[0];
+    const node = Object.values(item)[0];
```

Comment:
Afaik, this is the only part that maybe pushes up the min node version, since it requires at least node 7.5 according to [node.green](https://node.green). If node 6 is required, it's easy enough to rewrite this line to support node 6.

Reply:
Depending on what the min node version is expected to be, I can get rid of my flatmap implementation 😅

Reply:
Nvm. Realized we can skip this altogether.